ios/StripeTerminalReactNative.swift

```diff
@@ -49,6 +52,7 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
     var readReusableCardCancelable: Cancelable? = nil
     var cancelReaderConnectionCancellable: Cancelable? = nil
     var loggingToken: String? = nil
+    var terminal: Terminal? = nil
```

Comment:
Suggest removing this `terminal` and always using `Terminal.shared` to access the offline status.